core/templates/pages/exploration-editor-page/exploration-objective-editor/exploration-objective-editor.directive.html

```diff
@@ -25,4 +25,7 @@
   .modal-body exploration-objective-editor .form-text {
     margin-bottom: -8px;
   }
+  .oppia-form-text {
```

Comment:
Don't redefine classes. If this is commonly used, consider adding it to oppia.css and use it.